### DISCUSSION TITLE
Download facebook videos fail

### DISCUSSION THREAD

**Discussion by matthewpipes** [arabcoders/ytptube#137] [General] [answered]
Love what you've done here, great fork of metube. Just having trouble using it for facebook videos. Youtube videos worked fine. Heres my logs from docker. The GUI gives an error of "Failed to add link. The string did not match the expected pattern". Hoping you could help with this. If there's any other information needed let me know

ytptube  | 19:44:05 [DownloadQueue] [INFO ] Adding url='https://www.facebook.com/share/r/19oyCSLGb1/?mibextid=UalRPS' quality='best' format='any' folder=None output_template=None ytdlp_cookies='' ytdlp_config={}
ytptube  | 19:44:05 [aiohttp.server] [ERROR] Error handling request
ytptube  | Traceback (most recent call last):
ytptube  |   File "/opt/python/lib/python3.11/site-packages/aiohttp/web_protocol.py", line 477, in _handle_request
ytptube  |     resp = await request_handler(request)
ytptube  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ytptube  |   File "/opt/python/lib/python3.11/site-packages/aiohttp/web_app.py", line 559, in _handle
ytptube  |     return await handler(request)
ytptube  |            ^^^^^^^^^^^^^^^^^^^^^^
ytptube  |   File "/app/app/main.py", line 276, in add
ytptube  |     status = await self.add(
ytptube  |              ^^^^^^^^^^^^^^^
ytptube  |   File "/app/app/main.py", line 769, in add
ytptube  |     status = await self.queue.add(
ytptube  |              ^^^^^^^^^^^^^^^^^^^^^
ytptube  |   File "/app/app/DownloadQueue.py", line 264, in add
ytptube  |     return await self.__add_entry(
ytptube  |            ^^^^^^^^^^^^^^^^^^^^^^^
ytptube  |   File "/app/app/DownloadQueue.py", line 185, in __add_entry
ytptube  |     return await self.add(
ytptube  |                  ^^^^^^^^^
ytptube  | TypeError: DownloadQueue.add() got an unexpected keyword argument 'entry'

**Comment by arabcoders** [comment, marked as answer]
Hi, Thanks for the report. It has been fixed and changes should be live shortly.
